docs/install.rst

```diff
@@ -27,3 +27,50 @@ After installing MXNet, you can install the GluonNLP toolkit by
 
    pip install gluonnlp
 
+
+Install from Master Branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are intersted in trying out features on master branch that hasn't been released yet, you have
+the option of installing from master branch directly.
+
+
+Install from Github
++++++++++++++++++++
+
+Use the following command to automatically download and install the current code on master branch
+
+.. code-block:: console
+
+   pip install https://github.com/dmlc/gluon-nlp/tarball/master
+
+
+Install from Source Code
+++++++++++++++++++++++++
+
+You can also first check out the code locally using git
+
+.. code-block:: console
+
+   git clone https://github.com/dmlc/gluon-nlp
+   cd gluon-nlp
+
+then use the provided `setup.py` to install into site-packages
+
+.. code-block:: console
+
+   python setup.py install
+
+
+.. note::
+
+   You may need to use `sudo` in case you run into permission denied error.
+
+
+Alternatively, you can set up the package with development mode, so that local changes are
+immediately reflected in the installed python package
+
+.. code-block:: console
+
+   python setup.py develop
+
```

Comment:
Shall we also add a link on how to install mxnet nightly if users are asked to upgrade mxnet? https://mxnet.incubator.apache.org/install/index.html?version=master&platform=Linux&language=Python&processor=GPU

Reply:
Added